src/App.js

```diff
@@ -3,10 +3,12 @@ import React from 'react';
 import Design from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Your calculator app is failing upon performing an operation involving the division operator. For instance, when I try to compute ` 9 / 9 ` it returns a blank screen instead of ` 1 `, unlike performing a calculation with the other operators. Kindly ensure to improve upon this aspect.

Reply:
@rloterh Hi I realized that and fixed it like 20 minutes ago

Reply:
@rloterh if you check my last commit it is working

Reply:
@fed1k
Actually, no further changes need to be committed/ pushed changes once your project is submitted for review, except after the review is complete. Else, it is difficult for reviewers to notice the later changes. I apologize, I actually couldn't notice your changes as I happen to have cloned your project for review before you updated your code.

Reply:
@fed1k 
In case you've implemented the requested changes, kindly go ahead to submit your project for a second review.
Cheers! 😃

Reply:
Ohh okay I got it. That is not problem I will just remove commented code and resubmit again. Thanks for reviewing. Have a nice time.

Reply:
@rloterh Thanks again for the code review